testsuite/thermo_integration/test_thermo_integration.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import espressopp
+import mpi4py.MPI as MPI
+
+import unittest
+
+class TestThermoIntegration(unittest.TestCase):
+    def setUp(self):
+
```

Comment:
I guess @MrTheodor  refers to the empty line between the first imports (sys, time, etc.) and "import unittest" otherwise I don't see a problem with it.